ruleset.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<ruleset name="Centreon Standard">
+  <description>Custom rules defined by Centreon</description>
+
+  <exclude-pattern>centreon-build</exclude-pattern>
+  <exclude-pattern>GPL_LIB</exclude-pattern>
```

Comment:
Why exclude _GPL_LIB_ and _tests_ ? It seems that code within these directories is owned by the team and should therefore be accounted for.